package.json

```diff
@@ -13,7 +13,7 @@
     "testing",
     "api",
     "REST",
-    "jasmine",
+    "jest",
```

Comment:
I believe jasmine was replaced with jest

Reply:
Yes, thank you. This looks like a hold-over from v0.x 😱